version.js

```diff
@@ -0,0 +1 @@
+exports.version = "17.1.0";
```

Comment:
what is this file for?

Reply:
this file was need by pre-commit hook, I will probably remove it when implement versioning